recipe/meta.yaml

```diff
@@ -77,7 +85,7 @@ outputs:
 about:
   home: http://darwinsys.com/file
   summary: Fine Free File Command
-  license: BSD-2-Clause
+  license: BSD-2-Clause-Darwin
```

Comment:
Note this is the correct SPDX license identifier. It's a identifier specifically for `file`.
https://spdx.org/licenses/BSD-2-Clause-Darwin.html

The linter warns about it as it appears to be more recently added to SPDX. I opened a PR to smithy to update the SPDX license list.
https://github.com/conda-forge/conda-smithy/pull/2177